orders-war/src/main/resources/applicationContext.xml

```diff
@@ -88,9 +88,12 @@
         <property name="packagesToScan" value="au.gov.qld.pub.orders.entity"/>
     </bean>
  
-    <bean id="transactionManager" class="org.springframework.orm.jpa.JpaTransactionManager"/>
+    <bean id="transactionManager" class="org.springframework.orm.jpa.JpaTransactionManager">
+        <property name="dataSource" ref="dataSource"/>

```

Comment:
reduce confusion in latest spring, spring-boot may be better in the future than using xml configuration